lib/log_ignored_message/log_ignored.rb

```diff
@@ -0,0 +1,62 @@
+module LogIgnoredMessage
+  class LogIgnored
+    extend Configure::Macro
+    include Log::Dependency
+    include Dependency
+
+    def self.inherited(cls)
+      cls.class_exec do
+        dependency_module = Module.new do
+          define_singleton_method :included do |reciever_class|
+            reciever_class.class_exec do
+              ::Dependency::Attribute.define(self, :log_ingored, cls)
+
+              define_method :log_ingored do
+                @log_ingored ||= cls.configure self
+              end
```

Comment:
```suggestion
          define_singleton_method :included do |receiver_class|
            receiver_class.class_exec do
              ::Dependency::Attribute.define(self, :log_ignored, cls)

              define_method :log_ignored do
                @log_ignored ||= cls.configure self
              end
```

Reply:
I've removed this dependency for now. We can put it back in later if needed